tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md

```diff
@@ -0,0 +1,159 @@
+positive
+
+## Memulai Installasi
+Duration: 0:01
+
+Setelah konfigurasi penyimpanan. klik tombol 'Install sekarang'. Jendela kecil akan muncul dengan petinjauan opsi penyimpanan kamu pilih, dengan memungkinkan pergi kembali jika ada detail yang salah.
+
+Klik `Lanjutkan` untuk membenarkan perubahan di tempat dan memulai proses installasi.
+
+![screenshot](https://assets.ubuntu.com/v1/a76b68ce-ubuntu-verify-install.png)
+
+## Pilih lokasi kamu
+Duration: 0:01
+
+Jika kamu terkoneksi dengan internet, lokasi kamu akan terdeteksi automatis. Cek lokasi kamu jika benar dan klik 'Meneruskan'  untuk memproses.
+
+Jika kamu tidak yakin dengan zona waktu kamu, tulis nama kota lokal atau kota atau gunakan map untuk memilih lokasi.
+
+![screenshot](https://assets.ubuntu.com/v1/a942aa3d-download-desktop-install-ubuntu-desktop_6.jpg)
+
+positif
+: Jika kamu memiliki masalah mengkoneksikan ke Intenet, gunakan menu di tangan-atas-kanan sudut untuk memilih jaringan.
+
+## Pilih preferensi tata letak keyboard
+Duration: 0:01
+
+Untuk memanggil Ubuntu tata letak keyboard kamu, klik di pengaturan bahasa kamu butuh mengikuti tata letak spesifik keyboard kamu. Jika kamu tidak yakin, klik tombol '**Deteksi Tata Letak Keyboard**' untuk bantuan.
+
+Tata letak keyboard bisa selalu berubah, dan tata letak tambahan ditambahkan, dari desktop setelah instalasi.
+
+![screenshot](https://assets.ubuntu.com/v1/310b5196-download-desktop-install-ubuntu-desktop_7.jpg)
+
+## Informasi Login
+Duration: 0:01
+
+Masukkan nama kamu dan pemasang akan automatis mensugestikan nama komputer dan username. Ini bisa berubah dengan mudah jika kamu lebih suka. Nama komputer adalah bagaimana komputer akan muncul di jaringan, Username kamu akan menjadi login kamu dan nama akun.
+
+Selanjutnya, masukkan password yang kuat. Pemasang akan memberitahu jika itu sangat lemah.
+
+Kamu bisa memilih automatis login dan enskripsi folder rumah. Jika mesin kamu portable, kami merekomendasikan tetap automatis login dimatikan atau menyalakan enskripsi. Ini akan menghentikan orang lain mengakses file personal kamu jika mesin hilang atau dicuri.
+
+
+![screenshot](https://assets.ubuntu.com/v1/69d32ea7-download-desktop-install-ubuntu-desktop_8.jpg)
+
+: If enable home folder encryption and you forget your password, you won't be able to retrieve any personal data stored in your home folder.
+
+## Installasi di latar belakang
+Duration: 0:20
+
+Pemasang akan selesai di latar belakang ketika jendela pemasangan mengajari kamu sedikit tentang bagaimana mengagumkannya Ubuntu. Bergantung kecepatan mesin kamu dan koneksi jaringan, ini hanya akan memakan waktu beberapa menit.
+
+![screenshot](https://assets.ubuntu.com/v1/4e133117-download-desktop-install-ubuntu-desktop_9.jpg)
+
+## Installasi selesai
+Duration: 0:01
+
+Setelah semuanya terpasang dan terkonfigurasi, jendela kecil akan menampilkan pertanyaan untuk melakukan restart mesin. Klik `Restart Now` dan hapus diantara DVD atau drive USB.
```

Comment:
remove either the DVD or USB flash drive when promoted -> keluarkan DVD atau lepaskan flash disk USB ketika diminta
(promoted di sini kelihatannya salah ketik, dugaan saya 'prompted')